.github/workflows/scripts/kubernetes/core/run-quickstart-test-on-kubernetes.sh

```diff
@@ -130,35 +104,15 @@ runPostHelmInstallCommands
 echo "running the tests"
 pwd
 
-route="localhost:8080"
```

Comment:
I"m not sure I like 'hiding' this in the overridable-functions.

I think it would be cleaner to have some kind of check that can be overridden per tests in overridable-functions.sh, and an if/else check here. In the normal (if) case, run the tests normally in this file, and if we should run the tests ourselves implement that in the tests overridable-functions.sh.

BTW, I think a few other quickstarts use client/server, so if that can be a common use case which can be handled here, even better. Although perhaps that can come later.